playground/didkey/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "didkey"
+version = "0.1.0"
+authors = ["Jonas Johan Solsvik <jonasjso@protonmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+did-key = "*"
```

Comment:
Er dette didkit sim pakke did key?

Reply:
Nei, e en standalone pakke https://crates.io/crates/did-key

>This crate is intended to provide basic support for did:key methods. It has no external dependencies and can be compiled for any target. It was originally designed for use with DIDComm Extension for gRPC, but we recognized it may be useful if this was an independent library.